testing/code/test_excinfo.py

```diff
@@ -755,14 +764,18 @@ def f():
         assert tw.lines[0] == "    def f():"
         assert tw.lines[1] == ">       g(3)"
         assert tw.lines[2] == ""
-        assert tw.lines[3].endswith("mod.py:5: ")
```

Comment:
@nicoddemus Yes, otherwise the tests would fail. After I used `write` to print out the highlighted file path, the tests complained that there's no `write` method on the `TWMock` object. So I added a `write` method for `TWMock` and the corresponding way to check `write`s.

I think there must be a better way to do this, maybe I can refactor these tests and eliminate replace `TWMock` with a `mock.Mock` later in another pull request?